routes/licenses.js

```diff
@@ -1,20 +1,28 @@
 const Joi = require('joi');
 
+const errors = require('../services/util/errors');
 const { license: serialize } = require('../services/util/serializers');
 
 const routes = [];
 
 const licenseSchema = Joi.object({
   code: Joi.string().required(),
   name: Joi.string().required(),
-  url: Joi.string()
-    .uri()
-    .required(),
+  url: Joi.string().required(),
   groups: Joi.array()
     .required()
     .items(Joi.string()),
 });
 
+function handleError(h, { message }) {
+  switch (message) {
+    case errors.licenseNotFound:
+      return h.error(message, { statusCode: 404 });
```

Comment:
I implemented this similar to the 404-error response of the `/attribution` route.